vite.config.ts

```diff
@@ -0,0 +1,105 @@
+import { extname, resolve } from "path";
+import { defineConfig, loadEnv, UserConfig } from "vite";
+import viteCompression from "vite-plugin-compression";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+import {
+  injectCanisterIdPlugin,
+  preRenderAboutPlugin,
+  stripInjectJsScript,
+} from "./vite.plugins";
+
+const defaultConfig = (mode?: string): Omit<UserConfig, "root"> => {
+  const envPrefix = "II_" as const;
+
+  // Expand environment - .env files - with canister IDs
+  process.env = {
+    ...loadEnv(mode ?? "development", process.cwd(), envPrefix),
+    ...process.env,
+  };
+
+  return {
+    envDir: "../../",
+    publicDir: "assets",
+    envPrefix,
+    resolve: {
+      // TODO: use aliases for imports
+      alias: {
+        $assets: resolve(__dirname, "src/frontend/assets"),
+        $app: resolve(__dirname, "src/frontend/src"),
+        $generated: resolve(__dirname, "src/frontend/generated"),
+      },
+    },
+    build: {
+      outDir: "../../dist",
+      emptyOutDir: true,
+      rollupOptions: {
+        input: {
+          main: resolve(__dirname, "src/frontend/index.html"),
+          about: resolve(__dirname, "src/frontend/about.html"),
```

Comment:
should this be set based on `mode`? you seem to differentiate in the rest of the config

Reply:
`about.html` removed so this does not exist anymore